packages/jellyfish-api-core/__tests__/container_adapter_client.ts

```diff
@@ -31,7 +31,7 @@ export class ContainerAdapterClient extends ApiClient {
     })
 
     if (error != null) {
-      throw new RpcApiError({ ...error, rpcMethod: method })
+      throw new RpcApiError({ ...error, method: method })
```

Comment:
you can change `method: method` to `method`

Reply:
Will do it next time, this is required for Jellyfish -> Whale -> Playground -> Wallet. Each having 30+ minutes build and publish time.